apps/stream-deck/package.json

```diff
@@ -17,7 +17,8 @@
     "@types/node": "20.8.10",
     "rollup": "^4.0.2",
     "tslib": "^2.6.2",
-    "typescript": "^5.2.2"
+    "typescript": "^5.2.2",
+    "rollup-plugin-circular-dependencies": "^1.1.2"
```

Comment:
remove it, I want to see the circular dependency if it is there, even if I cannot do something about it.